libs/src/services/ethContracts/serviceProviderFactoryClient.js

```diff
@@ -25,7 +24,7 @@ class ServiceProviderFactoryClient extends ContractClient {
     if (!Utils.isFQDN(endpoint)) {
       throw new Error('Not a fully qualified domain name!')
     }
-    if (!Number.isInteger(amount) && !BN.isBN(amount)) {
+    if (!Number.isInteger(amount) && !Utils.isBN(amount)) {
```

Comment:
i like this solution since we already have a static web3 in Utils, but jw is there a libs situation in which we can't get a web3 instance from ethweb3manager?

Reply:
I think we should always be able to do that as well. But we don't need an instance of web3, we just need the static utils, so i think this is better